config/samples/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+## Append samples you want in your CSV to this file as resources ##
+resources:
+- saas_v1_apicast.yaml
```

Comment:
Is this `v1` correct?

I've seen that all Custom Resources stays on v1alpha1, I wonder if having here `v1` can affect to the kustomize boilerplate

Reply:
I missed that during the rollback! The idea was move to v1, but I rolled back after some discussion but missed that!

Fixed in https://github.com/3scale/saas-operator/pull/51/commits/707e58ef224c1f96456d0d616454c7d2e2f7c51e